### ISSUE TITLE
Release schedule hinting

### ISSUE THREAD

**Issue by farski** [syndicated-media/sn-spec#46]

**Comment by farski** [comment]
Somewhat related to #21, but more generalized. Some shows release automatically the same time every week; it would be good to have a way of telling aggregators that, so they can refresh the feed soon after.

iCalendar has fairly robust coverage of that kind of recurrence http://www.kanzaki.com/docs/ical/rrule.html

**Comment by Cj-Malone** [comment]
I was just thinking about this, but based on past updates. A defined way would be better and the RRULE looks perfect for this.

**Comment by farski** [comment]
I think most people do built in some sort of heuristic scheduling based on previous release times; that's what I've always done at least and it works out pretty well, but that is just an implementation thing.

The counter to this is that everyone should support PuSH or similar, so polling should be an exception, and it may be odd to have a standard for edge cases. Though for apps that hit feeds directly this could be potentially useful.

**Comment by cqr** [comment]
This is definitely worth doing. PuSH is not an option without a centralized server, which many clients can't or don't want to use.

**Comment by kookster** [comment]
doesn't this already exist?
http://web.resource.org/rss/1.0/modules/syndication/

**Comment by farski** [comment]
@kookster that really only handles periodic releases. It wouldn't really help if Serial want's to indicate their next episode is coming out in three months

**Comment by kookster** [comment]
@farski The initial description of this issue is that "Some shows release automatically the same time every week", so my understanding is that this issue was about hinting predictable recurrence, as opposed to #21 which speaks to hinting the next episode regardless of regular schedule.

**Comment by farski** [comment]
@kookster as whoops, got my tickets confused. There are a bunch of fairly common release cycles that I think can't be represented by syndication. Releasing on Tuesday at noon and Thursday at 5pm, for example. I may just not have looked at `sy:` hard enough to figure out how to make that happen

**Comment by cqr** [comment]
@kookster yeah, I think this might be a good place for syndicated.media to point people, if it feels like it will meet all of our needs. Worth pointing out that very few feeds implement this correctly right now. The 99pi feed, for example claims:
```
    <sy:updatePeriod>hourly</sy:updatePeriod>
    <sy:updateFrequency>1</sy:updateFrequency>
```

**Comment by kookster** [comment]
re: current adoption and use, I believe "frequency of update" does not equal "frequency of new episodes released", so it has been used as a hint for how often a feed would like to be polled for any kind of possible change to anything in the feed, rather than an indication of how often new episodes drop. For that reason alone, it may be a bad fit for this ticket's purpose.

As you can tell, I definitely don't think that `http://purl.org/rss/1.0/modules/syndication/` solves all problems, but for this ticket, seemed like it should at least be discussed.

Because it supports a base datetime, it can be used to indicate updates occur at a specific time each week, but not 2 different days and times in the same week (unless it was ever 60 hours or something odd like that, which I can't imagine someone actually specifying)

**Comment by Cj-Malone** [comment]
@kookster [Syndication](http://web.resource.org/rss/1.0/modules/syndication/) says updateBase uses yyyy-mm-ddThh:mm but the example is yyyy-mm-ddThh:mmZ. Do you know if they have decided which to use? And howcome the spec wasn't updated?
